deployment/ansible/app.yml

```diff
@@ -9,9 +9,9 @@
 
   roles:
     - { role: "ckan.app" }
-    - { role: "ckanext-disqus" }
-    - { role: "ckanext-spatial" }
-    - { role: "ckanext-datajson" }
-    - { role: "ckanext-odp_theme" }
-    - { role: "ckanext-googleanalytics" }
+    # - { role: "ckanext-disqus" }
+    # - { role: "ckanext-spatial" }
+    # - { role: "ckanext-datajson" }
```

Comment:
I tested it out and found that we can safely uncomment the `datajson` role, as it is now on the latest version and compatible with our new version of CKAN.

Reply:
Good call.  Done.